pkg/asset/machines/worker.go

```diff
@@ -88,7 +88,7 @@ func (w *Worker) Generate(dependencies asset.Parents) error {
 	}
 
 	ic := installconfig.Config
-	pool := workerPool(ic.Machines)
+	pool := workerPool(ic.Compute)
```

Comment:
This can be:

```go
for _, pool := range ic.Compute {
```

couldn't it?

Reply:
Ultimately, yes. But baby steps. And the validator only accepts a single compute machine pool named "worker" at the moment anyway.